.github/workflows/check-public-api.yaml

```diff
@@ -0,0 +1,60 @@
+name: Check public API
+
+env:
+  RUST_BACKTRACE: 1
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      # this is _not_ a regex, see: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+      - '[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
This should probably be removed, if a tag has been pushed to master by that point it's "too late". We should think about how to integrate this manual check into the release workflow.

Reply:
I removed it in the latest changeset.